### PR TITLE
new translations

### PR DESCRIPTION
**Pull request by Tasm-Devil**

### CODE REVIEW

File: locale/de_DE.po [hunk 1]
```diff
@@ -1349,148 +1347,148 @@ msgstr ""
...
 
 #: py/moduerrno.c:152
 msgid "Unsupported operation"
-msgstr ""
+msgstr "Nicht unterstützter Vorgang"
 
 #: py/moduerrno.c:153
 msgid "Invalid argument"
-msgstr ""
+msgstr "Ungültiges Argument
```
Comment (Tasm-Devil) [inline locale/de_DE.po]: " missing

Comment (Tasm-Devil) [reply locale/de_DE.po]: Fixed it.

**Comment by Tasm-Devil** [comment]
It finally passed. Next time, I will check for error before making a pull request. Online-editing is obviously not the way to make valid changes.

**Comment by tannewt** [comment]
@Tasm-Devil Its fine to rely on Travis. That's why it's there!

**Pull request merged**